server/routes/paymentPlan.routes.js

```diff
@@ -0,0 +1,85 @@
+import express from 'express';
+import { authenticate, schemaValidation, isAdmin, hasValidObjectId } from '../helpers/middleware';
+import addPlanSchema from '../schemas/paymentPlan.schema';
+import PaymentPlanController from '../controllers/paymentPlan.controllers';
+
+const router = express.Router();
+
+/**
+ * @swagger
+ * /payment/add:
```

Comment:
Please fix for other endpoints too.
```suggestion
 * /payment-plan/add:
```